qutebrowser/mainwindow/tabbedbrowser.py

```diff
@@ -110,30 +110,31 @@ class TabbedBrowser(tabwidget.TabWidget):
     new_tab = pyqtSignal(browsertab.AbstractTab, int)
 
     def __init__(self, *, win_id, private, parent=None):
-        super().__init__(win_id, parent)
+        super().__init__(parent)
+        self.widget = tabwidget.TabWidget(win_id, parent)
```

Comment:
Objects in Qt have [a parent set](http://doc.qt.io/qt-5/objecttrees.html) which helps when cleaning things up - here, you pass TabbedBrowser's parent as parent to TabWidget, so the TabWidget is a sibling of TabbedBrowser.

However, it'd make more sense if TabWidget was a child of TabbedBrowser, i.e. you should pass `parent=self` here.